src/software/ai/hl/stp/tactic/dribble/dribble_fsm.cpp

```diff
@@ -90,6 +90,36 @@ void DribbleFSM::getPossession(const Update &event)
                               event.common.world.field()) +
         Vector::createFromAngle(face_ball_orientation).normalize(0.05);
 
+    // if close to ball, pivot smartly
+    if (distance(event.common.robot.position(), ball_position) <
+        ROBOT_MAX_RADIUS_METERS * 5)
```

Comment:
can we make the dribble pivot distance to be an optional param that we pass into the event